fixed typos, examples, added descriptions for samples

d45 opened the pull request:
Removed a couple of code snippets in the docs that still had "tableau.addIn" 
Removed the Known Issues link that was broken (Known issues moved to the CC site).

sdesmond46 commented on docs/trex_examples.md:
```diff
@@ -3,9 +3,53 @@ title: Dashboard Extension Samples
+**In this section**
+
+* TOC
+{:toc}
```
What's this supposed to be?

d45 replied:
Ah, that's the kramdown macro that adds a table of contents at the top of a page. It grabs all the level 1, 2, .. etc. headings that are on the page puts in links. You can configure it in the _config.yml file to only grab certain headings.   I learned about the macro from Jared.

d45 replied:
Only works when you have GitHub pages.

sdesmond46 commented on docs/trex_examples.md:
```diff
@@ -24,12 +68,35 @@ Every Tableau extension has a manifest file (`.trex`) that describes the extensi
+
+The samples are set up so that the web server is using port `8765`.  If you need to specify a different port instead of `8765`, you can change ports using Python, if you have Python installed, or you can run another Node.js command.  
+
+From the `extensions-api` folder, start an HTTP server on port \<PORT\>, using one of the following commands:
```
For the port section, maybe put it in the inline code thing like `PORT`

d45 replied:
Good idea. Someone might think they need to include the angle brackets.

d45 commented:
I changed it to `PORT` and reworked the sentence.  
Ran spell check over the docs files.   
Changed the text that appears in the browser tab (now Extensions API Home)
